thetis/coupled_timeintegrator_2d.py

```diff
@@ -65,10 +65,7 @@ def _create_swe_integrator(self):
                 semi_implicit=self.options.timestepper_options.use_semi_implicit_linearization,
                 theta=self.options.timestepper_options.implicitness_theta)
         else:
-            self.timesteppers.swe2d = self.swe_integrator(solver.eq_sw, self.fields.solution_2d,
-                                                          fields, solver.dt,
-                                                          bnd_conditions=solver.bnd_functions['shallow_water'],
-                                                          solver_parameters=self.options.timestepper_options.solver_parameters)
+            self.timesteppers.swe2d = self.swe_integrator(solver.eq_sw, self.fields.solution_2d, fields, solver.dt, bnd_conditions=solver.bnd_functions['shallow_water'], solver_parameters=self.options.timestepper_options.solver_parameters)
```

Comment:
I'd split it visually as the lines deleted

Reply:
I agree

Reply:
I have changed this and committed it to the branch